src/ProductList/Home.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import ProductList from './ProductList'
+import "../css/Home.css";
+import faker from "faker";
+import { useCart } from '../context/CartContext';
+
+faker.seed(123);
+export const products = [...Array(50)].map((item) => ({
+  id: faker.random.uuid(),
+  name: faker.commerce.productName(),
+  image: faker.random.image(),
+  price: faker.commerce.price(),
+  material: faker.commerce.productMaterial(),
+  brand: faker.lorem.word(),
+  inStock: faker.random.boolean(),
+  fastDelivery: faker.random.boolean(),
+  quantity:faker.random.number({
+    'min': 1,
+    'max': 1
+}),
+  ratings: faker.random.arrayElement([1, 2, 3, 4, 5]),
+  offer: faker.random.arrayElement([
+    "Save 50",
+    "70% bonanza",
+    "Republic Day Sale"
+  ]),
+  idealFor: faker.random.arrayElement([
+    "Men",
+    "Women",
+    "Girl",
+    "Boy",
+    "Senior"
+  ]),
+  level: faker.random.arrayElement([
+    "beginner",
+    "amateur",
+    "intermediate",
+    "advanced",
+    "professional"
+  ]),
+  color: faker.commerce.color()
+}));
+
+function Home() {
+  const [state,dispatch]=useCart();
+  const getProductsUnderPrice = (products, priceRange) => {
+    if (priceRange) return products.filter((item) => item.price <= priceRange);
+    return products;
```

Comment:
```suggestion
    if (priceRange) {
    return products.filter((item) => item.price <= priceRange);
    }
    return products;
```
Just for better readability...I think Tanay sir mentioned this... please correct me if i am wrong.